### ISSUE TITLE
Maven dependency resolution exception (com.github.noraui:ojdbc7:jar:12.1.0.2)

### ISSUE THREAD

**Issue by cosmiccreepers** [antidot/db2triples#25]
Hello,

I'm coming across the following dependency error when trying to compile with Maven. 

The summary of it:
```
[ERROR] Failed to execute goal on project db2triples: Could not resolve dependencies for project net.antidot:db2triples:jar:2.1-SNAPSHOT: Could not find artifact com.github.noraui:ojdbc7:jar:12.1.0.2 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
```

Including other warnings:
```
[WARNING]
[WARNING] Some problems were encountered while building the effective model for net.antidot:db2triples:jar:2.1-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-gpg-plugin is missing. @ line 171, column 15
[WARNING] 'build.plugins.plugin.version' for org.apache.maven.plugins:maven-failsafe-plugin is missing. @ line 159, column 16
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
[WARNING] The project net.antidot:db2triples:jar:2.1-SNAPSHOT uses prerequisites which is only intended for maven-plugin projects but not for non maven-plugin projects. For such purposes you should use the maven-enforcer-plugin. See https://maven.apache.org/enforcer/enforcer-rules/requireMavenVersion.html
[INFO]
[INFO] -----------------------< net.antidot:db2triples >-----------------------
[INFO] Building DB2Triples 2.1-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-failsafe-plugin/maven-metadata.xml
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-failsafe-plugin/maven-metadata.xml (1.3 kB at 3.6 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-gpg-plugin/maven-metadata.xml
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/maven/plugins/maven-gpg-plugin/maven-metadata.xml (626 B at 10 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/github/noraui/ojdbc7/12.1.0.2/ojdbc7-12.1.0.2.pom
[WARNING] The POM for com.github.noraui:ojdbc7:jar:12.1.0.2 is missing, no dependency information available
Downloading from central: https://repo.maven.apache.org/maven2/com/github/noraui/ojdbc7/12.1.0.2/ojdbc7-12.1.0.2.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.264 s
[INFO] Finished at: 2018-12-11T11:36:40+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project db2triples: Could not resolve dependencies for project net.antidot:db2triples:jar:2.1-SNAPSHOT: Could not find artifact com.github.noraui:ojdbc7:jar:12.1.0.2 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

**Comment by ailintom** [comment]
I also had this issue. Oracle Driver is not in the maven repository due to license restrictions. A simple workaround would be to remove ojdbc7 from dependencies if you do not need Oracle. Simply remove
`    <dependency>
    	<groupId>com.github.noraui</groupId>
    	<artifactId>ojdbc7</artifactId>
    	<version>12.1.0.2</version>
    	<type>jar</type>
    	<scope>runtime</scope>
    </dependency>`
from pom.xml. Otherwise you can add ojdbc7 to your local maven repository

**Comment by psass-antidot** [comment]
Fixed in #27
